### ISSUE TITLE
Private networks not working

### ISSUE THREAD

**Issue by berendt** [boxcutter/fedora#1]
At the moment it is not possible to use private networks with this box (Fedora 21) and current Vagrant.

```
==> default: Configuring and enabling network interfaces...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

ARPCHECK=no /sbin/ifup eth1 2> /dev/null

Stdout from the command:

ERROR    : [/etc/sysconfig/network-scripts/ifup-eth] Device eth1 does not seem to be present, delaying initialization.


Stderr from the command:
```

```
[vagrant@openstack ~]$ ip a s
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default 
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: enp0s3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether 08:00:27:03:50:41 brd ff:ff:ff:ff:ff:ff
    inet 10.0.2.15/24 brd 10.0.2.255 scope global dynamic enp0s3
       valid_lft 86274sec preferred_lft 86274sec
    inet6 fe80::a00:27ff:fe03:5041/64 scope link 
       valid_lft forever preferred_lft forever
3: enp0s8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether 08:00:27:61:ad:8d brd ff:ff:ff:ff:ff:ff
    inet6 fe80::a00:27ff:fe61:ad8d/64 scope link 
       valid_lft forever preferred_lft forever
```

In `dmesg`:

```
[    4.571002] e1000 0000:00:08.0 enp0s8: renamed from eth1
[    4.658664] e1000 0000:00:03.0 enp0s3: renamed from eth0
```

```
[vagrant@openstack ~]$  ll /etc/sysconfig/network-scripts/ifcfg-*
-rw-r--r-- 1 root root 168 Apr 14 09:54 /etc/sysconfig/network-scripts/ifcfg-enp0s3
-rw-r--r-- 1 root root 633 Apr 14 09:54 /etc/sysconfig/network-scripts/ifcfg-eth1
-rw-r--r-- 1 root root 254 Apr 14 09:54 /etc/sysconfig/network-scripts/ifcfg-lo
```

Makefile.local:

```
CM := nocm
HEADLESS := true
UPDATE := true
PACKER := /opt/packer/packer
PACKER_DEBUG := true
```

Vagrantfile:

```
...
config.vm.network :private_network, ip: '10.25.25.25'
...
```

```
% vagrant version
Installed Version: 1.7.2
Latest Version: 1.7.2

You're running an up-to-date version of Vagrant!
```

**Comment by deromemont** [comment]
+1

**Comment by marun** [comment]
Fedora support in 1.7.2 is broken.  I'm running from master and boxes built from these recipes works fine.  I recommend running from source until such time as 1.7.3 is released.

**Comment by berendt** [comment]
This has to be re-tested with the latest released version 1.7.4. I will take care of this tomorrow.

**Comment by annawake** [comment]
Fedora 21 EOL
